cmd/gateway-gcs.go

```diff
@@ -80,8 +80,13 @@ func gcsMultipartMetaName(uploadID string) string {
 }
 
 // Returns name of the part object.
-func gcsMultipartDataName(uploadID, etag string) string {
-	return fmt.Sprintf("%s/%s/%s", gcsMinioMultipartPathV1, uploadID, etag)
+func gcsMultipartDataName(uploadID string, partNumber int, etag string) string {
+	return fmt.Sprintf("%s/%s/%05d.%s", gcsMinioMultipartPathV1, uploadID, partNumber, etag)
+}
+
+// Returns name of the composed object.
+func gcsMultipartComposeName(uploadID string, composeNumber int) string {
```

Comment:
As this function is used only in `CompleteMultipartUpload()`, is it good idea to make it as closure?